package.json

```
@@ -8,57 +8,76 @@
     "lib"
   ],
   "dependencies": {
-    "accepts": "^1.3.3",
+    "accepts": "^1.3.8",
     "animated-gif-detector": "^1.2.0",
-    "exif-reader": "^1.0.2",
+    "exif-reader": "^2.0.0",
     "gm-papandreou": "^1.23.0-patch1",
     "hijackresponse": "^5.0.0",
-    "httperrors": "^2.0.1",
-    "impro": "~0.14.1",
-    "inkscape": "^3.0.0",
+    "httperrors": "^2.3.0",
+    "impro": "~0.14.2",
+    "inkscape": "^3.1.1",
     "jpegtran": "^2.0.0",
-    "mime": "^3.0.0",
+    "mime": "=3.0.0",
     "optimist": "^0.6.1",
-    "optipng": "^4.0.0",
+    "optipng": "^4.2.0",
     "passerror": "^1.1.1",
     "pngcrush": "^3.0.0",
-    "pngquant": "^4.0.0",
-    "sharp": "^0.32.0"
+    "pngquant": "^4.2.0",
+    "sharp": "^0.33.1"
   },
   "devDependencies": {
-    "browser-sync": "^2.18.6",
-    "compression": "^1.6.1",
-    "eslint": "^8.1.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-config-standard": "^17.0.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-mocha": "^10.0.1",
-    "eslint-plugin-n": "^15.1.0",
-    "eslint-plugin-node": "^11.0.0",
-    "eslint-plugin-promise": "^6.0.0",
+    "browser-sync": "^3.0.2",
+    "compression": "^1.7.4",
+    "eslint": "^8.56.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-config-standard": "^17.1.0",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-mocha": "^10.2.0",
+    "eslint-plugin-n": "^16.6.2",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^6.1.1",
     "eslint-plugin-standard": "^5.0.0",
-    "express": "^4.16.3",
-    "gifsicle": "^5.3.0",
-    "magicpen": "^6.0.2",
-    "magicpen-prism": "^5.0.0",
+    "express": "^4.18.2",
+    "gifsicle": "=5.3.0",
```

Same?

---

Pinned it with commit https://github.com/papandreou/express-processimage/commit/441b0821bc3e6be1a6ee80dccf61042814df96dd

The newer versions are using `"type": "module"` and to get those to work would lead to bigger refactoring. Hence marked the pinning explicitly.

---

The ESM rewrite landed in 6.0.0, so we should keep using an open range in case fixes get backported:

```suggestion
    "gifsicle": "^5.3.0",
```